### ISSUE TITLE
Add support for quad curves in font writing

### ISSUE THREAD

**Issue by aui** [opentypejs/opentype.js#84]
trueType use `toBuffer ()` and `download ()` method error.

**BUG DEMO**: 

http://aui.github.io/demo/opentype.js/readwrite-ttf.html

**Comment by zswang** [comment]
from https://raw.githubusercontent.com/DmitryBaranovskiy/raphael/master/raphael.js
```
 q2c = function (x1, y1, ax, ay, x2, y2) {
            var _13 = 1 / 3,
                _23 = 2 / 3;
            return [
                    _13 * x1 + _23 * ax,
                    _13 * y1 + _23 * ay,
                    _13 * x2 + _23 * ax,
                    _13 * y2 + _23 * ay,
                    x2,
                    y2
                ];
        }
```

**Comment by fdb** [comment]
This is fixed with #86, thanks @zswang!
